templates/index.html

```diff
@@ -1,692 +1,425 @@
-                      <h3>Quick to install</h3>
-                      <p>Snaps are quick to install, easy to create, safe to run, and they
-                      update automatically and transactionally so your app is always fresh and
-                      never broken.</p>
-                      <p><a href="/docs/core/install">Install snapd&nbsp;&rsaquo;</a></p>
-                  </div>
-                  <div class="equal-height--vertical-divider__item four-col">
-                      <h3>Work anywhere</h3>
-                      <p>Snaps aim to work on any distribution or device, from IoT devices to
-                      servers, desktops to mobile devices.</p>
-                      <p><a href="https://github.com/snapcore/snapd" class="external">Build snapd from source</a></p>
-                  </div>
-                  <div class="equal-height--vertical-divider__item four-col last-col">
-                      <h3>GitHub and beyond</h3>
-                      <p>The public collection of snaps includes the best of GitHub and beyond,
-                      so you have the whole world of Linux at your fingertips.</p>
-                      <p><a href="https://build.snapcraft.io" class="external">Build a snap from GitHub</a></p>
-                  </div>
-                </div>
-            </div>
+        <nav class="p-navigation__nav" role="menubar">
+          <span class="u-off-screen">
+            <a href="#main-content">Jump to main content</a>
+          </span>
+          <ul class="p-navigation__links" role="menu">
+            <li class="p-navigation__link" role="menuitem"><a href="https://build.snapcraft.io">Build</a></li>
+            <li class="p-navigation__link" role="menuitem"><a href="/docs">Docs</a></li>
+            <li class="p-navigation__link" role="menuitem"><a class="p-link--external" href="https://tutorials.ubuntu.com">Tutorials</a></li>
+            <li class="p-navigation__link" role="menuitem"><a class="p-link--external" href="https://forum.snapcraft.io/categories">Forum</a></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <div class="p-strip--accent is-deep snapcraft-hero">
+      <h1 class="u-off-screen">Snapcraft</h1>
+      <div class="row">
+        <div class="col-12">
+          <p class="p-heading--two">Package, distribute, and update any app for Linux and IoT</p>
+          <p class="p-heading--five">Snaps are containerised software packages that are simple to create and install. They auto-update and are safe to run and because they bundle their dependencies, they work on all major Linux systems without modification.</p>
+          <a href="https://tutorials.ubuntu.com/tutorial/create-your-first-snap" class="u-width-auto p-button--positive is-inline--right">Build your first snap</a>
+          <span class="is-inline--right">or</span>
+          <a href="https://build.snapcraft.io" class="u-width-auto p-button--neutral">Get started with GitHub</a>
```

Comment:
Would you consider putting the button section in a `p` tag, for example:
```html
<p>
    <a href="https://tutorials.ubuntu.com/tutorial/create-your-first-snap" class="p-button--positive is-inline--right">Build your first snap</a> or <a href="https://build.snapcraft.io" class="p-button--neutral">Get started with GitHub</a>
</p>
```

Reply:
@anthonydillon I removed the wrapping p around the buttons as it was causing a duplicated margin-top: p+p (1.5rem) + *+* (2rem) on the two anchors inside. I know u-no-margin--top can address that, but then on mobile it leads to the two buttons (or the text "or" in between) to get too close to each other like this:
![image](https://user-images.githubusercontent.com/2741678/32567673-e0015e5e-c4b3-11e7-883d-7dd680a78d5b.png)

Curious though, since a button is meant to look and behave like a single line block, while a paragraph is something that naturally wraps, is there a reason it needs to be inside a p?

Reply:
The p is for proper spacing but I see you would get margin top on the second button. It was just a suggestion. Feel free to leave it as it is.